### DISCUSSION TITLE
Clear feature suggetion

### DISCUSSION THREAD

**Discussion by kaje0105** [variar/klogg#361] [Ideas]
I wish there was a function to clear the contents of the search results window at the bottom of the program window.
When debugging a program, it is often re-executed several times, and in this case, it is confusing to see the previous search results together.

Anyway, this program is very useful to me.
Thank you so much.

**Comment by variar** [comment]
Could you describe your use case a little bit more? It seems that a file with debug output of some program is opened in klogg, and when program is re-executed this file is overwritten. klogg reloads this file, but search results are not cleared?

**Reply by kaje0105** [reply]
OK.
For example, my log file is appended real-time.
I opened my log file with klogg and inserted a filter to see only lines that match a specific word.

It is OK. works fine.

But I want to clear matches. 
Because I want to see the match result from now, not the match result from the past.

Is it possible now?
Thanks.

**Reply by variar** [reply]
It is possible to set a line from where the search for matches will be performed.

**Reply by kaje0105** [reply]
I mean, I want to clear below section. (red mark)
![image](https://user-images.githubusercontent.com/34637947/128315239-a38bb294-4656-4933-9a2d-c03eb8d68080.png)

**Reply by variar** [reply]
Yes, I understand. Using search limits it should be possible to achieve desired behavior.  Search limits allow you to set the part of file, where filtering should work. In example case, setting search start to some line will cause all next searches to ignore file content before the selected line.

Unfortunately, there seems to be a bug with search limits in auto-refresh mode #375
